tests/gtest/avifavmtest.cc

```diff
@@ -53,9 +53,9 @@ TEST_P(AvmTest, EncodeDecode) {
     decoder->codecChoice = av1_codec;
     // AVIF_RESULT_NO_CODEC_AVAILABLE is expected because av1_codec is not
     // enabled or because we are trying to decode an AV2 file with an AV1 codec.
-    ASSERT_EQ(avifDecoderReadMemory(decoder.get(), decoded.get(), encoded.data,
+    ASSERT_NE(avifDecoderReadMemory(decoder.get(), decoded.get(), encoded.data,
```

Comment:
These tests are broken in the main branch, but because of a typo in the github actions workflow it was never being executed. dav1d in particular was returning `AVIF_RESULT_DECODE_COLOR_FAILED`, not `AVIF_RESULT_NO_CODEC_AVAILABLE`.

Reply:
This is now in #1748